queries/liquidsoap/highlights.scm

```diff
@@ -0,0 +1,66 @@
+(op) @keyword.operator
+"..." @keyword.operator
+
+[
+  "open"
+  "def"
+  "let"
+  "if"
+  "then"
+  "elsif"
+  "else"
+  "begin"
+  "end"
+  "while"
+  "do"
+  "to"
+  "for"
+  "fun"
+  "try"
+  "catch"
```

Comment:
these two could be `@exception`

Reply:
Is `@exception` for raising calls? These are control statements, ended with and `end` so they should probably have the same color through and through?

The raising statement is `error.raise` but it can be overriden by a local variable. How does `locals` interact with `highlights` in the situations?

Reply:
Nevermind, I applied it and checked it looks great. Fixed!